addons/mail/static/src/components/thread_preview/thread_preview.js

```diff
@@ -108,7 +108,9 @@ class ThreadPreview extends Component {
      * @param {MouseEvent} ev
      */
     _onClickMarkAsRead(ev) {
-        this.thread.markAsSeen();
+        if (this.thread.lastMessage) {
```

Comment:
Just to note it somewhere, we should probably hide/disable the mark as read button if there's no message.

Reply:
https://www.odoo.com/web#id=2336398&action=333&active_id=1750&model=project.task&view_type=form&cids=1&menu_id=4720